v_3_3_1/master_template.go

```diff
@@ -642,7 +642,7 @@ write_files:
             - sh
             - -c
             - sysctl -w net.core.somaxconn=32768; sysctl -w net.ipv4.ip_local_port_range="1024 65535"
-            image: alpine:3.6
+            image: quay.io/giantswarm/alpine:3.6
```

Comment:
fwiw 3.7 is out, do we stick to 3.6 for any reason?

Reply:
I guess we could use this to upgrade here, also for security, WDYT?

Reply:
:+1: fine with me